### DISCUSSION TITLE
New Blog Post: "How to Set up Netlify Deploys for Docfx using GitHub Actions"

### DISCUSSION THREAD

**Discussion by SeanKilleen** [dotnet/docfx#9711] [Show and tell]
Thought folks might be interested in how we're doing this on the NUnit project.

https://seankilleen.com/2024/02/how-to-set-up-netlify-deploys-for-docfx-using-github-actions/

**Comment by yufeih** [comment]
Thank you for sharing this informative blog post. This will certainly be beneficial for many working on similar NUnit projects. Looking forward to learning more from your future posts.

**Reply by SeanKilleen** [reply]
Thanks for taking such good care of a nice library!
